### ISSUE TITLE
Unable to interract with the interactive prompts in shell

### ISSUE THREAD

**Issue by akash07k** [microsoft/terminal#13810]
### Windows Terminal version

Microsoft.WindowsTerminalPreview version 1.15.2003.0

### Windows build number

10.0.25182

### Other Software

NVDA 2022.2 Stable and latest Alpha, WSL2

### Steps to reproduce

1. Start NVDA.
2. Open windows terminal.
3. Open a shell (WSL can also be used)
4. now type sudo apt install exim4 openssl ca-certificates -y and press enter.
5. Now type sudo dpkg-reconfigure exim4-config
6. Interractive prompt will be launch which will ask some questions for configuring exim4.
7. Press the arrow keys and observe. (We are unable to know relyably that which option is got focused because the announcement of the focus is very much confusing)
8. Now switch to some other window by pressing ALT+Tab.
9. Come back to the terminal window and try to read via object nav commands. (Most of the times NVDA won't read anything).
10. Alternatively, `timedatectl list-timezones` can also be tried, this issue persists with every such prompt

### Expected Behavior

Screen reader should announce the selected options properly.

### Actual Behavior

Screen reader behaves very unrelyably on the interractive prompts and we can't determine properly that which item is focused.
Also, choosing the "OK" and "Cancel" options is very difficult.
Please check below speech output:
Akash@pi: ~
speak command keys on
NVDA+down arrow
Bottom
NVDA+down arrow
Bottom
NVDA+up arrow
NVDA+up arrow
NVDA+up arrow
NVDA+t
akash@pi: ~
NVDA+t
akash@pi: ~
NVDA+up arrow
NVDA+down arrow
Bottom
NVDA+down arrow
Bottom
NVDA+t
akash@pi: ~
down arrow
    └──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────┘    
down arrow
    │                                          mail sent by smarthost; no local mail                                                                   │    
down arrow
    │                                          local delivery only; not on a network                                                                   │    
down arrow
    │                                          no configuration at this time                                                                           │    
up arrow
    │                                          no configuration at this time                                                                           │    
up arrow
    │                                          no configuration at this time                                                                           │    
up arrow
    │                                          local delivery only; not on a network                                                                   │    
up arrow
    │                                          mail sent by smarthost; no local mail                                                                   │    
up arrow
    │                                          mail sent by smarthost; received via SMTP or fetchmail                                                  │    
down arrow
    │                                          mail sent by smarthost; received via SMTP or fetchmail                                                  │    
down arrow
    │                                          mail sent by smarthost; received via SMTP or fetchmail                                                  │    
down arrow
    │                                          mail sent by smarthost; no local mail                                                                   │    
down arrow
    │                                          local delivery only; not on a network                                                                   │    
down arrow
    │                                          no configuration at this time                                                                           │    
down arrow
    │                                          no configuration at this time                                                                           │    
down arrow
    │                                          no configuration at this time                                                                           │    
up arrow
    │                                          no configuration at this time                                                                           │    
up arrow
    │                                          no configuration at this time                                                                           │    
up arrow
    │                                          local delivery only; not on a network                                                                   │    
up arrow
    │                                          mail sent by smarthost; no local mail                                                                   │    
up arrow
    │                                          mail sent by smarthost; received via SMTP or fetchmail                                                  │    
tab
tab
tab
tab
NVDA+up arrow

Please see Same issue logged with [NVDA](https://github.com/nvaccess/nvda/issues/14042)

**Comment by codeofdusk** [comment]
This affects terminal both with and without UIA notifications. I imagine this may be hard to solve/require a lot of thought as these are tough in any terminal.

CC @carlos-zamora.

**Comment by zadjii-msft** [comment]
Isn't this _basically_ the same as #13666/?

**Comment by carlos-zamora** [comment]
Yeah, I'm gonna say this is a duplicate of #13666.

/dup #13666
